### ISSUE TITLE
find a way to generate table with data for each month

### ISSUE THREAD

**Issue by JakubJKrawczyk** [JakubJKrawczyk/ProgramSaldo-TheWayOfDapper#7]

**Comment by Lakasabasz** [comment]
You mean copy table with data?

**Comment by JakubJKrawczyk** [comment]
Yes and format it into data table in app. Including check boxes, combo boxes and colours.

**Comment by Lakasabasz** [comment]
I can create new function `copyTable` to copy all structure and data, but everything you need to format, you will do it yourself. It'll be done in next sprint.
